arviz/plots/traceplot.py

```diff
@@ -137,6 +140,9 @@ def plot_trace(
         divergence_data = False
 
     data = get_coords(convert_to_dataset(data, group="posterior"), coords)
+
+    data = transform(data)
```

Comment:
By doing this you are basically writing `data = None(data)`, which is not legal Python. Instead you should do something like:

```python
if transform is not None:
    data = transform(data)
```

Reply:
@aloctavodia Sorry for this elementary mistake! I fixed it.